lib/util/mongo.js

```diff
@@ -170,17 +170,27 @@ function getPrimary(params, cb) {
 }
 
 function hasUserSpaceDb() {
-  if (process.env.MONGODB_USERDB_NAMESPACE) {
+  if (process.env.MONGO_CONN_STRING || process.env.MONGODB_USERDB_NAMESPACE) {
```

Comment:
@odra we are no longer going to use this env var name, should we revert this to using the service name which will be on of` mongodb-1` or `mongodb-1,mongodb-2,mongodb-3`

Reply:
This change is being addressed in this pr: #99